tests/gui/test_updater.py

```diff
@@ -165,8 +168,10 @@ def test_user_prompts(
     # Check disabling update checks.
     prompt_mock().launch.return_value = False  # type: ignore [attr-defined]
     expected_settings["updater_check"] = False
-    assert updater.should_check_for_updates() == False
-    assert updater.dangerzone.settings.get_updater_settings() == expected_settings
+
+    with qtbot.waitSignal(updater.update_check_toggled) as blocker:
+        assert updater.should_check_for_updates() == False
+        assert updater.dangerzone.settings.get_updater_settings() == expected_settings
```

Comment:
I totally agree with having a test, but I think that this diff does not end up checking the status of the `toggle_updates_action` checkbox.